src/components/Text.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+const Text = ({ size = "medium", color = "#000", children }) => {
+  const fontSizes = {
```

Comment:
One downside to this approach is that if I passed an unknown size such as. "shmedium", then my size would be undefined.